src/utils/utils.ts

```diff
@@ -0,0 +1,25 @@
+export const extractTokenAndUserId = (url: string) => {
```

Comment:
@CptHash @Trushy @louischristner @Iboakg5745 I would like you guys to review this little piece of code. I've made this function to extract the userId and the token from the url sent to my mailbox in order to change my password. But I'm pretty sure that I've done it in a bad way as you can see, I didn't figure out the exact regex to extract the token and the userId at once from the url, that's why I would to have you guys review my code, thank you.